frontend/src/static/js/components/styles/ManageItemList.scss

```
@@ -118,45 +118,28 @@
 	margin: 0;
 	border-style: solid;
 	border-width: 0 0 1px;
-	border-color: #f0f0f0;
+	border-color: var(--border-subtle);
 
-	background-color: var(--user-action-form-inner-bg-color);
-
-	.dark_theme & {
-		border-color: #2d2d2d;
-	}
+	background-color: var(--bg-surface, var(--user-action-form-inner-bg-color));
```

Stylelint (17.11.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)